### ISSUE TITLE
SideNav fixed breaks when window is maximized

### ISSUE THREAD

**Issue by rahimi-ali** [Dogfalo/materialize#6400]
I have a fixed SideNav which is always shown on large and is triggered by a button in small or mid just like the example in [materializecss](https://materializecss.com/sidenav.html#variations) The problem is if I resize my browser to mobile size open the sidenav with the button and then use the maximize button of browser the grey overlay stays on the page and cant be closed.

The Situation:
<img width="960" alt="problem" src="https://user-images.githubusercontent.com/33478509/60092033-2fb94580-975b-11e9-92a0-84ab2f0ce57a.png">

The Code:  
````
<!DOCTYPE html>
  <html lang="en">
  <head>
      <meta charset="UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="ie=edge">
      <title>Document</title>
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
  </head>
  <body>
  
  <ul id="filter-sort-menu" class="sidenav sidenav-fixed">
      <ul>
          <li><a href="#!">First Sidebar Link</a></li>
          <li><a href="#!">Second Sidebar Link</a></li>
        </ul>
    </ul>
    <a href="#" id="filter-sort-btn" data-target="filter-sort-menu" class="sidenav-trigger btn-floating"><i class="material-icons">search</i></a>
  </body>
    <!-- Compiled and minified JavaScript -->
    <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"> </script>
    <script>
        $(function () {
            $("#filter-sort-menu").sidenav({
                closeOnClick: true,
                edge: 'right'
            });
        })
    </script>
````

By the way this happens on both Chrome 75 and Firefox 67

@Dogfalo

**Comment by anchit1501** [comment]
Fixed #6400

**Comment by anchit1501** [comment]
Closes #6400
